### ISSUE TITLE
Update MD-RAID template README with instructions on using it in passive mode

### ISSUE THREAD

**Issue by danboid** [radiorabe/rabe-zabbix#91]
I have successfully got your MD-RAID template to work under Zabbix 4.0 but I had to change a few settings to use passive instead of active monitoring. I think the MD-RAID README should be updated with instructions on how to do this because its not at all obvious (to Zabbix newbs) and it took me a while to work it out / remember how to do it.

There may be a better way to do it but I used these steps (after following the existing installation instructions):

Under the Zabbix web interface go to Configuration -> Hosts -> then click 'Discovery' next to the relevant host. Now click 'Item prototypes' next to the 'RAID device (MD) discovery' rule then click on each of the 5 item prototypes and change the Type of all 5 Raid device item prototypes from 'Zabbix agent (active)' to 'Zabbix agent'.

**Comment by tschaerni** [comment]
Strange... I'm also using it under 4.0, since some months and didn't have to change anything. I also applied it some days ago to a new host, it's just working.

Maybe it's because all of my hosts go over zabbix proxies?

**Comment by danboid** [comment]
Maybe that's the reason? I'm not using any proxies, just a single zabbix 4.0 server.

**Comment by danboid** [comment]
In that case, maybe update the README something like

"If you aren't getting any data from this template, try changing it to work in passive mode by..."

**Comment by danboid** [comment]
Another possibility is I have misconfigured my server but I think it would be handy to document how to adjust this template to use passive checks instead anyway.

**Comment by danboid** [comment]
Ah!

I hadn't uncommented / defined RefreshActiveChecks on the server using this template. Do Active checks not work without this being set? I'm definitely a Zabbix newb.

**Comment by danboid** [comment]
No, it's not the lack of RefreshActiveChecks. I deleted the MD-RAID using host, added RefreshActiveChecks (using the default value) to the agent config, restarted the agent and then re-added the host back into Zabbix and its still not showing any MD-RAID data.

**Comment by danboid** [comment]
It seems it was because RefreshActiveChecks was commented that this template wasn't working for me. I deleted the host, uncommented RefreshActiveChecks from the agent config, restarted the agent and then re-added the host into Zabbix and its working now.

**Comment by danboid** [comment]
Oh dear!

I'm not sure myself now. I thought that deleting a host from Zabbix would reset the template settings for that host but I've just checked the item prototypes and they're all set to Zabbix agent, not Zabbix agent (active) and I'm not sure who to reset them. Maybe I'd need to remove the template entirely?

**Comment by danboid** [comment]
Maybe its a bug in Zabbix but it seems that when you've changed the type of an Item prototype to Zabbix agent, the type drop-down disappears from that screen so you can't revert it to Zabbix agent (active), unless I'm missing something?

**Comment by paraenggu** [comment]
@danboid Zabbix agent items can either be used in [passive or active mode](https://www.zabbix.com/documentation/3.0/manual/concepts/agent#passive_and_active_checks) depending on your setup preferences or network policies. We've [decided to use active checks for our templates](https://github.com/radiorabe/rabe-zabbix#further-template-conventions).

> I thought that deleting a host from Zabbix would reset the template settings for that host but I've just checked the item prototypes and they're all set to Zabbix agent, not Zabbix agent (active) and I'm not sure how to reset them.

Did you change the items on the host level or on the template directly? I assume the later, therefore deleting the host and all items won't have any impact. If you would like to use passive items, delete the imported template (to get a clean environment), re-import it and change all zabbix items from active to passive.
